### PR TITLE
Tooltip component, a react-tooltip wrapper (#938)

### PR DESCRIPTION
**Pull request by Lowhearth**
Hello! 

I've been trying to solve #938 and came up with a possible solution:

Screenshot 2019-04-27 at 19 31 53
Screenshot 2019-04-27 at 19 31 46
Screenshot 2019-04-27 at 19 31 36

This component will allow to render a react-tooltip component easily at the body level (as recommended in the react-tooltip documentation) with literally no effort.

I'd like to know what are your thoughts about it 

Thanks!

### CODE REVIEW

**Comment by jacogr** [comment]
Like the approach, fits in with what we are doing elsewhere.

File: packages/ui-app/src/Tooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import { BareProps } from './types';
+
+const rootElement = document.getElementById('tooltips');
+
+type Props = BareProps & {
+  trigger?: string,
+  delayShow?: number,
+  place?: 'bottom' | 'top' | 'right' | 'left',
+  effect?: 'solid' | 'float'
+
```
Comment (jacogr) [inline packages/ui-app/src/Tooltip.tsx]: (nitpick) extra blank line is not needed here

File: packages/ui-app/src/Tooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+};
+
+export default class Tooltip extends React.PureComponent<Props> {
+
```
Comment (jacogr) [inline packages/ui-app/src/Tooltip.tsx]: (nit) Extra unneeded blank line

File: packages/ui-app/src/Tooltip.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  static defaultProps = {
+    trigger: 'default-trigger',
+    delayShow: 250,
+    effect: 'solid',
+    place: 'bottom',
+    className: 'ui--Tooltip'
+  };
+
+  tooltipContainer: HTMLElement;
+  constructor (props: Props) {
```
Comment (jacogr) [inline packages/ui-app/src/Tooltip.tsx]: (nitpick) Prefer a blank line just before

Comment (Lowhearth) [reply packages/ui-app/src/Tooltip.tsx]: Thanks for the tips I'm trying to improve my code formatting skills :)

Comment (Lowhearth) [reply packages/ui-app/src/Tooltip.tsx]: Making the "trigger" prop mandatory will reflect better the default implementation of the component, so I just updated it in the same commit, hope you don't mind !

Thanks!

**Pull request merged**